src/control-plane/backend/lambda/sfn-workflow/index.ts

```diff
@@ -228,7 +228,7 @@ async function _getStackFromS3(region: string, bucket: string, prefix: string, s
 async function _getOutputsFromS3(region: string, bucket: string, prefix: string, stackName: string) {
   const fileKey = `${prefix}/${stackName}/output.json`;
   try {
-    const content = await _getStackFromS3(region, bucket, fileKey, stackName);
+    const content = await _getStackFromS3(region, bucket, fileKey);
```

Comment:
remove the `stackName` from the function

Reply:
`const fileKey = `${prefix}/${stackName}/output.json`;`